stdlib/config-api/src/main/java/org/ballerinalang/stdlib/config/GetConfig.java

```diff
@@ -32,19 +30,13 @@
  *
  * @since 0.970.0-alpha3
  */
-@BallerinaFunction(
-        orgName = "ballerina", packageName = "config",
-        functionName = "get"
-)
 public class GetConfig {
-
     private static final ConfigRegistry configRegistry = ConfigRegistry.getInstance();
-    private static final String lookupErrReason = "{ballerina/config}LookupError";
-
-    public static Object get(Strand strand, String configKey, Object type) {
+    private static final String LOOKUP_ERROR_REASON = "{ballerina/config}LookupError";
```

Comment:
Spotbugs will complain here. Only public constants should be in all caps.

Reply:
@pubudu91 We have used ALL_CAPS for private static final constants in other places too. I don't think it will complain.